tests/server/wildfly/pom.xml

```diff
@@ -98,16 +92,9 @@
     </dependency>
 
     <dependency>
-      <groupId>org.apache.camel</groupId>
-      <artifactId>camel-example-servlet-rest-tomcat</artifactId>
-      <type>war</type>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.jboss.byteman</groupId>
-      <artifactId>byteman</artifactId>
-      <scope>test</scope>
+      <groupId>com.squareup.okhttp</groupId>
+      <artifactId>okhttp</artifactId>
+      <version>${version.com.squareup.okhttp}</version>
```

Comment:
Versions should be in parent pom.

Ok to use this dependency for tests, but before using it elsewhere we would need to check if it was ok for product support.